<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>332</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Connect to Server</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="ServerNameLabel_2">
      <property name="styleSheet">
       <string notr="true">font-size: 30px;
font-family: &quot;Times New Roman&quot;, Times, serif;</string>
      </property>
      <property name="text">
       <string>SQL Server Connect</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="ServerNameLabel">
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
font-family: &quot;Times New Roman&quot;, Times, serif;</string>
          </property>
          <property name="text">
           <string>Server name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="LogLabel">
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
font-family: &quot;Times New Roman&quot;, Times, serif;</string>
          </property>
          <property name="text">
           <string>Login:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="PassLabel">
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
font-family: &quot;Times New Roman&quot;, Times, serif;</string>
          </property>
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="NamedbLabel">
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
font-family: &quot;Times New Roman&quot;, Times, serif;</string>
          </property>
          <property name="text">
           <string>Name DB:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="ServerNameEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="LogEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="PassEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="NamedbEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>267</x>
     <y>300</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="Connect">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
